### DISCUSSION TITLE
GameObject.CompareTag(tag) 함수가 Android에서 동작하지 않습니다

### DISCUSSION THREAD

**Discussion by Cha-SeongHun** [naverz/zepeto-studio-kor#465] [Issue Report (Legacy)]
아래는 제가 사용한 코드의 일부입니다.
```ts
private OnTriggerEnter(coll: Collider) {
    if (coll.CompareTag("Item")) {
      this.StartCoroutine(this.OnHitToybox(coll.gameObject));
    }
  }
```

이 코드가 PC 유니티 에디터에서 테스트 할 떄에는 정상적으로 동작하지만
Android에 올려 테스트 시 CompareTag함수가 동작하지 않습니다.
대신 ``` coll.tag == "Item" ``` 을 사용시 모바일에서도 정상 동작하는 것은 확인하였습니다.

제 폰 기종은 갤럭시 폴드2입니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 아이폰에서는 CompareTag코드가 정상적으로 동작하는 것을 확인하였습니다.
안드로이드에서도 확인해보고 말씀 드리겠습니다.

**Reply by Cha-SeongHun** [reply]
넵 확인 감사드립니다🙂
모바일 환경은 질문에 언급된것처럼 폴드2이며, QR코드를 통해 월드에 접속하였습니다

**Reply by Zepeto-TechEvangelist** [reply]
World 1.1.5 안드로이드 S10에서 QR빌드 결과 CompareTag 정상 작동 되는 점 확인했습니다.
만약 계속 안되신다면 coll.tag == "Item"로 사용하시는 것도 방법입니다.

**Reply by kyn320** [reply]
폴드3에서도 CompareTag가 작동하지 않습니다. 확인 부탁드립니다 : )
